### ISSUE TITLE
Exceptions or multiple visible windows?

### ISSUE THREAD

**Issue by dynek** [dwarvesf/blurred#22]
### Describe the bug

Not a bug. I would like to ask if you think you will ever implement exceptions or multiple visible windows?
I would like to be able to one specific window constantly visible, and regular behavior for others.

Thank you for that great piece of software!

### Steps to reproduce

N/A

### Expected behaviour

N/A

### Environment

OS X Monterey

### Logs / Stack trace

N/A

**Comment by jakobmann** [comment]
Agree. It would also be nice to have an option to keep the top windows for each screen unblurred. Great tool this.

**Comment by Simon-CR** [comment]
I'm also looking for a way to have exception, windows that you never want blurred.  otherwise good stuff
